Assets/Scripts/Auction/BiddingRound.cs

```diff
@@ -11,6 +12,8 @@ public class BiddingRound
     [SerializeField] 
     public PlayerManager[] players;
 
+    private Dictionary<PlayerManager, int> chipsInPlay = new Dictionary<PlayerManager, int>();
```

Comment:
Why PlayerManager and not PlayerIdentity here?

Reply:
The inclusion of this change was accidental, as I was experimenting with integrating the existing BiddingRound. BiddingRound is currently deprecated, only being left in case we want to record bidding outcomes for post-match viewing. I do agree that it should be PlayerIdentity, but as of now, it was used to not have to refactor the entire previous auctionDriver code which was using playerManager instead of identity